app/src/components/AddFeedbackForm/index.js

```diff
@@ -0,0 +1,161 @@
+  item.touched && item.error;
+
+class AddFeedbackForm extends Component {
+  constructor() {
+    super();
+    this.formIsInvalid = this.formIsInvalid.bind(this);
+  }
+  formIsInvalid() {
+    const {
+      nameInput,
+      descriptionInput,
+      urlInput,
+    } = this.props;
+    return itemInvalid(nameInput) ||
+      itemInvalid(descriptionInput) ||
+      itemInvalid(urlInput);
+  }
+  render() {
+    const {
+      nameInput,
+      urlInput,
+      descriptionInput,
+      onSubmitFeedback,
+      onClear,
+      user,
+      message,
+    } = this.props;
+    return (
+      <div>
+        {message &&
+          <Heading
+            tag="h4"
+            align="center"
+          >
+            <Paragraph>
+              {message}
+            </Paragraph>
+          </Heading>
+        }
+      <Headline
+        size="small"
+        strong
+      >
+        We, Thank You For Your Feedback!
+      </Headline>
+      <Heading
+        tag="h4"
+        align="center"
+      >
+        Please Fill Out The Form Below...
+      </Heading>
+      <Form>
+        <FormFields>
+          <FormField
+            label="Name *"
+            htmlFor="feedbackNameInput"
+            help="Please make sure your name is correct or Edit your profile to correct it!"
+            error={nameInput.touched && nameInput.error ? nameInput.error : null}
+          >
+            <input
+              {...nameInput}
+              ref="nameInput"
+              className={styles.blockedInput}
+              id="feedbackNameInput"
+              type="text"
+              name="name"
+              className={styles.blockedInput}
+              disabled
+              required
+            />
+          </FormField>
+          <FormField
+            label="URL *"
+            htmlFor="feedbackUrlnput"
+            help="Fill in the URL you want to give feedback about"
+            error={urlInput.touched && urlInput.error ? urlInput.error : null}
+          >
+            <input
+              {...urlInput}
+              ref="urlInput"
+              className={styles.longUrl}
+              id="urlInput"
+              type="text"
+              name="url"
+              disabled
+              required
+            />
+          </FormField>
+          <FormField
+            label="Description"
+            htmlFor="feedbackDescriptionInput"
+            help="Describe your problem/feedback"
+            error={descriptionInput.touched && descriptionInput.error
+               ? descriptionInput.error : null
+            }
+          >
+            <textarea
+              {...descriptionInput}
+              id="feedbackDescriptionInput"
+              ref="feedbackDescriptionInput"
+              type="text"
+              rows="5"
+              cols="40"
+              required
+            />
+          </FormField>
+        </FormFields>
+        <Footer className={styles.footer}>
+          {user ?
+          <Menu direction="row">
```

Comment:
indent this block so that it's in by two spaces.  I think our ESLint will be mad about this.